modules/pipeline/providers/edgepipeline_register/interface.go

```diff
@@ -27,9 +28,43 @@ import (
 )
 
 type Interface interface {
+	GetAccessToken(req apistructs.OAuth2TokenGetRequest) (*apistructs.OAuth2Token, error)
+	GetOAuth2Token(req apistructs.OAuth2TokenGetRequest) (*apistructs.OAuth2Token, error)
+	GetEdgePipelineEnvs() apistructs.ClusterDialerClientDetail
+	CheckAccessToken(token string) error
 	//RegisterEdgeToDialer(ctx context.Context)
 }
 
+func (p *provider) GetAccessToken(req apistructs.OAuth2TokenGetRequest) (*apistructs.OAuth2Token, error) {
+	return &apistructs.OAuth2Token{
+		AccessToken: p.Cfg.accessToken,
+		ExpiresIn:   0,
+		TokenType:   "Bearer",
+	}, nil
+}
+
+func (p *provider) GetOAuth2Token(req apistructs.OAuth2TokenGetRequest) (*apistructs.OAuth2Token, error) {
```

Comment:
... req is not used?

Reply:
now, all edge pipeline, task will return the same token